server/db/driver/pg/accounts.go

```diff
@@ -164,13 +164,13 @@ out:
 	}
 
 	pubKeyBytes := childExtKey.SerializedPubKey()
-	addr, err := dcrutil.NewAddressSecpPubKey(pubKeyBytes, a.keyParams)
+	addr, err := stdaddr.NewAddressPubKeyEcdsaSecp256k1V0Raw(pubKeyBytes, a.keyParams)
```

Comment:
Need pubkey *hash* address (`NewAddressPubKeyHashEcdsaSecp256k1V0`).  Previously we were relying on the `Address` method in the `return` to canonicalize to pkh, but that's no longer the case. Thus we have to make the address the correct type here.


```diff
-       addr, err := stdaddr.NewAddressPubKeyEcdsaSecp256k1V0Raw(pubKeyBytes, a.keyParams)
+       addr, err := stdaddr.NewAddressPubKeyHashEcdsaSecp256k1V0(stdaddr.Hash160(pubKeyBytes), a.keyParams)
        if err != nil {
-               log.Errorf("error creating new AddressSecpPubKey: %v", err)
-               return "", fmt.Errorf("error encoding fee address")
+               log.Errorf("Failed to create creating new pubkey hash address: %v", err)
+               return "", fmt.Errorf("error encoding fee address: %w", err)
        }
```

Reply:
Also please apply the following diff to server/auth/registrar.go, otherwise it's difficult to trace this issue down:

```diff
diff --git a/server/auth/registrar.go b/server/auth/registrar.go
index 88f0dd87..0ffda883 100644
--- a/server/auth/registrar.go
+++ b/server/auth/registrar.go
@@ -4,6 +4,7 @@
 package auth
 
 import (
+       "errors"
        "fmt"
        "time"
 
@@ -11,6 +12,7 @@ import (
        "decred.org/dcrdex/dex/msgjson"
        "decred.org/dcrdex/dex/wait"
        "decred.org/dcrdex/server/account"
+       "decred.org/dcrdex/server/asset"
        "decred.org/dcrdex/server/comms"
 )
 
@@ -196,6 +198,10 @@ func (auth *AuthManager) handleNotifyFee(conn comms.Link, msg *msgjson.Message)
 func (auth *AuthManager) validateFee(conn comms.Link, acctID account.AccountID, notifyFee *msgjson.NotifyFee, msgID uint64, coinID []byte, regAddr string) bool {
        addr, val, confs, err := auth.checkFee(coinID)
        if err != nil || confs < auth.feeConfs {
+               if err != nil && !errors.Is(err, asset.CoinNotFoundError) {
+                       log.Warnf("Unexpected error checking fee coin confirmations: %v", err)
+                       // return wait.DontTryAgain // maybe
+               }
                return wait.TryAgain
        }
        var msgErr *msgjson.Error
```